bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r"Error in '.*': "
+                     r'Near line (?P<line>\d+) '
+                     r"\(last key parsed '.*'\): (?P<message>.*)")
+class TOMLBear:
+    """
+    Checks the code for formatting in ``TOML`` documents
+    and prints each key's type.
+
+    This is done using ``tomlv`` utility. For more information visit
+    <https://github.com/BurntSushi/toml/tree/master/cmd/tomlv>.
+    """
+    LANGUAGES = {'TOML'}
+    REQUIREMENTS = {
+        GoRequirement('github.com/BurntSushi/toml/cmd/tomlv', '0.1.0'),
+    }
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('-types', filename)
```

Comment:
`'-types'` here is unnecessary.  It causes tomlv to output more information, which the bear then discards.  Not helpful.